cloud-inquisitor/aws_cloudfront_resource.go

```diff
@@ -506,5 +560,41 @@ func (cf *AWSCloudFrontDistributionHijackableResource) PublishState() error {
 }
 
 func (cf *AWSCloudFrontDistributionHijackableResource) AnalyzeForHijack() (*model.HijackableResourceChain, error) {
-	return &model.HijackableResourceChain{}, nil
+	switch cf.EventName {
+	case "DeleteDistribution":
+		return cf.analyzedeleteDistributionEntries()
+	default:
+		return &model.HijackableResourceChain{}, nil
```

Comment:
Needs a check here to see if its possible to create a cloud front distribution with an origin that points to a resource that doesnt exist. If this is possible, we need to check for this.

Need to check here to see if it is possible to create a distro that point to a resource that exists but we dont own.

Both cases need to have analysis code.

This may just need to be its own ticket.